src/hrc1155.spec.ts

```diff
@@ -35,12 +36,14 @@ describe('HRC1155 Contract Interface', () => {
 
   describe.skip('balanceOf', () => {
     it('should get the number of tokens in the specified account with id as a number', async () => {
+      const mockedBalance = await FAKE_BALANCE_HRC1155
+      const stub = sinon.stub(contract, 'balanceOf').withArgs(TEST_ADDRESS_1, TOKEN_GOLD)
+      stub.resolves().returns(FAKE_BALANCE_HRC1155)
```

Comment:
here we are mocking the actual method being called, so this test won't test our functionality, but rather the sinon mock 😅. So, if we ever were to introduce a bug in our `balanceOf` method, we wouldn't catch it with the test.

What should be mocked is either the `call` method or the contract that lives inside the `BaseToken` instance.

Reply:
You're right!